packages/core/src/scripts/utils/create-symlinks.ts

```diff
@@ -26,7 +30,7 @@ export default async () => {
     dependencyNames.map(async (name) => {
       const packageDir = dependencies[name].replace(/^(file:)/, "");
 
-      const dir = await resolve(process.cwd(), packageDir);
+      const dir = resolve(process.env.CWD, packageDir);
```

Comment:
@iamuchejude I've changed this ⬆️ 

Because I set the `process.env.CWD` in the test I can "mock" that we are running the command in a fake project directory (which is really just a temp dir). But make sure that it still works when you test manually 😀

Reply:
Oh cool!